docs/concepts/services/storage-provider/file-picker-options.md

```diff
@@ -91,6 +91,7 @@ Avalonia has set of built-in file types:
 - FilePickerFileTypes.ImageAll - all images
 - FilePickerFileTypes.ImageJpg - jpg images
 - FilePickerFileTypes.ImagePng - png images
+- FilePickerFileTypes.ImageWebP - webp images
```

Comment:
To clarify, we have a MinVersion tag to indicate certain features are new. In particular, `<MinVersion version="11.1" />` for your case can be used to indicate that WebP has been newly added.

You could opt to make a separate, short section if you feel that's better. ie. `## WebP Options <MinVersion version="11.1" />` to clarify both `FilePickerTypes.ImageWebP` and the `Patterns` part are new. Ensure that users won't be mistaken that WebP loading/rendering support is new, only the picker type changes.

If the styling seems janky on the tag where you put it, let me know and I will fix the React component after merging.

Reply:
@stevemonaco I'm not sure if you're talking to me or @maxkatz6, but I'm happy to revise the PR according to the recommendations you outlined if you're talking to me.

Reply:
@Nicholas-Westby Yes, if you could revise that would be great.

Reply:
@stevemonaco OK, I have added a section explaining that the WebP changes apply to 11.1+, as well as some hints on how to accomplish similar on older versions using more custom code.

If you'd like more changes, feel free to edit for expedience, or to make further recommendations to me.

Reply:
LGTM. Thanks for the contribution.